wrappers/src/object_cs.cpp

```diff
@@ -280,11 +281,21 @@ extern "C" {
         });
     }
 
-    REALM_EXPORT ManagedNotificationTokenContext* object_add_notification_callback(Object* object, void* managed_object, NativeException::Marshallable& ex)
+    REALM_EXPORT ManagedNotificationTokenContext* object_add_notification_callback(Object* object, void* managed_object, size_t* property_indices, size_t property_count, NativeException::Marshallable& ex)
     {
+        TableKey tk = object->get_object_schema().table_key;
```

Comment:
As we discussed before, this probably will need to go in his separate method, but for now it's fine.